src/api/getters.js

```diff
@@ -0,0 +1,5 @@
+export const getTargetLemma = (word) => word.targets[0].lemma;
```

Comment:
I realize you're at the mercy of the API you're interfacing with and are in all cases just working with whatever the first target is, but out of curiosity: What other information does `targets` contain (if/when it has multiple values)?

Reply:
I'm not sure why, but `target` never actually has more than one value